### DISCUSSION TITLE
How to pass `None` to `tauri::api::dialog::blocking::message`?

### DISCUSSION THREAD

**Discussion by leejy12** [tauri-apps/tauri#7861] [Q&A] [answered]
Hi, my app starts up like this (simplified):

```rust
fn init() -> Result<(), ()> {
     ...
}

fn main() {
    match init() {
        Ok(_) => {
            tauri::Builder::default()
                .manage(...)
                .invoke_handler(tauri::generate_handler![...])
                .run(tauri::generate_context!())
                .expect("error while running tauri application");
        }
        Err(_) => tauri::api::dialog::blocking::message(None, "Error", "Camera connection failed!"),
    }
}
```

Basically, if `init()` fails, I just want to show a messagebox and exit. But `tauri::api::dialog::blocking::message` expects an `Option<&Window<R>>` type for its first paramter. What should I pass to this function? Or is there another way to achieve what I'm trying to do?

Thank you.

**Comment by FabianLars** [comment, marked as answer]
I think `None::<&tauri::Window>` should work. Or you could use the `rfd` rust crate directly as long as you open the dialog outside of the tauri app context. I _think_ that will actually be required on macos https://github.com/tauri-apps/tauri/issues/6301 🤔

**Reply by leejy12** [reply]
Thank you! That worked like a charm :)
